transport_nantes/mobilito/views.py

```diff
@@ -108,11 +107,17 @@ def get_seen_pages_from_cookie(self) -> list:
             if pickled_pages_seen:
                 pages_seen_list: list = pickle.loads(
                     b64decode(pickled_pages_seen))
+                if hasattr(self, 'this_page') and self.this_page not in pages_seen_list:
```

Comment:
If you passed `this_page` as an argument, you could just test if it's `None` rather than doing the somewhat mystical `has_attribute`.

Reply:
This is done